syntax-test_dev.py

```diff
@@ -44,6 +41,10 @@ def get_details_of_test_assertion_line(view, pos):
     - the assertion characters (2nd item in tuple aka `assertion_colrange`)
     """
 
+    AssertionLineDetails = namedtuple(
```

Comment:
I preferred this in the module root (since its instances are being used outside of the function).